src/test/scala/mesosphere/marathon/storage/repository/GcActorTest.scala

```diff
@@ -132,13 +134,13 @@ class GcActorTest extends AkkaUnitTest with TestKitBase with GivenWhenThen with
         f.actor.stateName should equal(Scanning)
         f.actor.stateData should equal(UpdatedEntities())
         scanSem.release()
-        processReceiveUntil(f.actor, Idle) should be(Idle)
+        processReceiveUntil(f.actor, ReadyForGc) should be(ReadyForGc)
```

Comment:
Isn't this just `eventually { f.actor.stateName should be(ReadyForGc) }`?

Reply:
I'm sorry but we should clean this up. I know that you did not write `processReceiveUntil`. However, we should follow the boy scouts rule here and use `eventually` instead.